### PR TITLE
Manual Dart SDK / Engine roll

### PR DESCRIPTION
**Pull request by bkonyi**
Manual roll to handle removal of the packagesPath parameter for gen_snapshot.

#28101 is redundant after this change.

### CODE REVIEW

File: bin/internal/engine.version [hunk 1]
```diff
@@ -1 +1 @@
-6d7eb52185b117a3972cac4e23625f97198114d9
+61fc1786fd98f48ee355f800cf5d7aef0fff985b
```
Comment (aam) [inline bin/internal/engine.version]: perhaps put eol back?

**Review by aam — APPROVED** [review]

**Comment by aam** [comment]
It looks like some further tweaks to build_test.dart are needed to remove packages checks

**Comment by dnfield** [comment]
Can you pull #28176 back into this?

**Comment by cbracken** [comment]
Superseded by #28269

**Pull request closed without merging**